prep_request.go

```diff
@@ -650,3 +651,131 @@ func EpollCtl(epfd int, op int, fd int, event *syscall.EpollEvent) PrepRequest {
 		)
 	}
 }
+
+func Mkdirat(dirFd int, path string, mode uint32) (PrepRequest, error) {
+	b, err := syscall.ByteSliceFromString(path)
+	if err != nil {
+		return nil, err
+	}
+
+	bp := unsafe.Pointer(&b[0])
+	return func(sqe *iouring_syscall.SubmissionQueueEntry, userData *UserData) {
+		userData.hold(&b)
+		userData.request.resolver = fdResolver
+
+		sqe.PrepOperation(
+			iouring_syscall.IORING_OP_MKDIRAT,
+			int32(dirFd),
+			uint64(uintptr(bp)),
+			mode,
+			0,
+		)
+	}, nil
+}
+
+func Unlinkat(fd int, path string, flags int32) (PrepRequest, error) {
+	b, err := syscall.ByteSliceFromString(path)
+	if err != nil {
+		return nil, err
+	}
+
+	bp := unsafe.Pointer(&b[0])
+	return func(sqe *iouring_syscall.SubmissionQueueEntry, userData *UserData) {
+		userData.hold(&b)
+		userData.request.resolver = fdResolver
+
+		sqe.PrepOperation(
+			iouring_syscall.IORING_OP_UNLINKAT,
+			int32(fd),
+			uint64(uintptr(bp)),
+			0,
+			0,
+		)
+		sqe.SetOpFlags(uint32(flags))
+	}, nil
+}
+
+func Symlinkat(target string, newDirFd int, linkPath string) (PrepRequest, error) {
+	bTarget, err := syscall.ByteSliceFromString(target)
+	if err != nil {
+		return nil, err
+	}
+	bLink, err := syscall.ByteSliceFromString(linkPath)
+	if err != nil {
+		return nil, err
+	}
+
+	bpTarget := unsafe.Pointer(&bTarget[0])
+	bpLink := unsafe.Pointer(&bLink[0])
+	return func(sqe *iouring_syscall.SubmissionQueueEntry, userData *UserData) {
+		userData.hold(&bTarget)
+		userData.Hold(&bLink)
```

Comment:
The `userData.hold` could set multiple parameters
```go
userData.hold(&bTarget, &bLink)
```

Reply:
Fixed on last commit :)